website/content/docs/job-specification/resources.mdx

```diff
@@ -86,5 +86,49 @@ resources {
   }
 }
 ```
+## Memory Oversubscription
+
+Setting task memory limits is a challenging act of balance. Job authors must set
+a memory limit for each task — if it’s too low, the task may be interrupted if
+it exceeds the limit unexpectedly; if it’s too high, the cluster is left
+underutilized and resources are wasted. Job authors usually set limits based on
+their task's typical memory usage and then add a safety margin to handle
+unexpected load spikes or uncommon scenarios. That means that in some clusters,
+20-40% of the cluster memory may be reserved but unused.
+
+To help minimize reserved but not used cluster memory, Nomad 1.1 lets job
+authors set two separate memory limits:
+
+* `memory`: the reserve limit to represent the task’s typical memory usage —
+  this number is used by the Nomad scheduler to reserve and place the task
+
+* `memory_max`: the maximum memory the task may use, if the client has excess
+  available memory, and may be terminated if it exceeds
+
+If a client's memory becomes contended or low, the operating system will
+pressure the running tasks to free up memory. If the contention persist, Nomad
+may kill oversubscriped tasks and reschedule them to other clients. The exact
+mechanism for memory pressure is specific to the task driver and application
+runtime.
+
+The new max limit attribute is currently supported by Docker and all default
+Nomad drivers except raw_exec and QEMU. Consult the third-party drivers
+documentation for memory oversubscription support.
+
+Nomad operators must enable [Memory Oversubscription in Scheduler
+Config](/api-docs/operator/scheduler#update-scheduler-configuration); memory
+oversubscription is opt-in. Enterprise customers can use [Resource
+Quotas](https://learn.hashicorp.com/tutorials/nomad/quotas) to limit the memory
+oversubscription.
+
+To avoid degrading the cluster experience, we recommend examining and monitoring
+resource utilization and considering the following suggestions:
+
+* Set `oom_score_adj` for Linux host services that aren't managed by Nomad, e.g.
+  Docker, logging services, and the Nomad agent itself. For Systemd services, you can use the [`OOMScoreAdj` field](https://github.com/hashicorp/nomad/blob/v1.0.0/dist/systemd/nomad.service#L25).
+
+* Monitor hosts for memory utilization and set alerts on Out-Of-Memory errors
+
+* Set the [client `reserved`](/docs/configuration/client#reserved) with enough memory for host unmanaged services as well as a buffer for the memory excess. e.g. If the client reserved memory is 1GB, the allocations on the host may exceeed their soft memory limit by almost 1GB in aggregate before the memory becomes contended and allocations get killed.
```

Comment:
The use of "e.g." here is technically correct but if you've got a full sentence of stuff after it, it reads nicer to make that a standalone sentence. Also cleaned up the "host unmanaged services" for clarity.

```suggestion
* Set the [client `reserved`](/docs/configuration/client#reserved) with enough
  memory for host services that aren't managed by Nomad as well as a buffer
  for the memory excess. For example, if the client reserved memory is 1GB,
  the allocations on the host may exceed their soft memory limit by almost
  1GB in aggregate before the memory becomes contended and allocations get
  killed.
```